ws-android/files/workspace/a02-create-gradle-android-apk.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash  
+
+#  only need to run this script with the command (do not type the #)
+#  bash a02-create-gradle-android-apk.sh
+#  best in cloud 9 to just right click this file and select run
+
+
+
+
+echo "Enter a folder name, no spaces, no dashes, underscore allowed, for your app and folder example: helloGradle"
+read wow4
+
+
+ls -l /home/ubuntu/workspace/android-sdk-linux/platforms
+
+echo "Type the number of the target your Android app is for"
+echo "See above folders of installed platforms"
+echo "Type 20 for --target android-20    Android 4.4W.2 KitKat, type 20"
+echo "Type 22 for --target android-22    Android 5.1.1  Lollipop, type 22"
+echo "Type 23 for --target android-23    Android 6.0    Marshmallow, type 23"
+
+read myNum
+
+
+#Good site to check
+#http://socialcompare.com/en/comparison/android-versions-comparison
+
+
+android create project \
```

Comment:
Check out ws-wordpress. We should create this project in the docker file. /home/ubuntu/workspace is overwritten with the contents of the git repo if you choose to clone one on creation, so the project creation is only there as a quick way to get going and/or a hello world type app for users to see how quick they can get going.